skipper.yaml

```diff
@@ -35,6 +35,9 @@ volumes:
   - /tmp:/tmp/
   - /dev/:/dev
   - /run/udev:/run/udev
+
+  # Network manager dnsmasq
+  - /etc/NetworkManager/dnsmasq.d:/etc/NetworkManager/dnsmasq.d
```

Comment:
Please add a comment on why we're mounting it

Reply:
OK